contrib/!source-control/github/packages.el

```diff
@@ -86,3 +87,25 @@
         "gfC" 'spacemacs/git-link-commit-copy-url-only)
       ;; default is to open the generated link
       (setq git-link-open-in-browser t))))
+
+(defun github/init-magit-gh-pulls ()
+  (use-package magit-gh-pulls
+    :commands magit-gh-pulls-mode
```

Comment:
The MELPA package comes with autoloads, which is not the case when this is an extension. So you can change it to `:defer t`.